charts/gsp-cluster/templates/02-gsp-system/cloudhsm-service.yaml

```diff
@@ -0,0 +1,88 @@
+{{ if .Values.global.cluster.cloudHSMExpose }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Release.Name }}-haproxy
```

Comment:
Could we name this something descriptive like `{{ .Release.Name }}-hsmproxy` ... otherwise it might be a bit generic sitting in the cluster with a name like "gsp-haproxy"

Reply:
Thanks, changed as suggested.